src/main/java/Duke.java

```diff
@@ -7,17 +7,26 @@ public static void main(String[] args) {
 //                + "| |_| | |_| |   <  __/\n"
 //                + "|____/ \\__,_|_|\\_\\___|\n";
 //        System.out.println("Hello from\n" + logo);
+        String[] tasks = new String[100];
```

Comment:
I think it would be easier to use an ArrayList

`import java.util.ArrayList;`

That way you don't need to keep track of the counter because ArrayList has a size() method